portal-kernel/src/com/liferay/portal/kernel/theme/packageinfo

```diff
@@ -1 +1 @@
-version 3.1.0
\ No newline at end of file
+version 4.0.0
```

Comment:
This scares me a little... let's see if Brian wants to up kernel. If not, we'll need to simply deprecate these.